src/Command/InstalledRepositoryLoadingCommandTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpcq\Command;
+
+use Phpcq\Exception\RuntimeException;
+use Phpcq\Platform\PlatformInformation;
+use Phpcq\Repository\InstalledRepositoryLoader;
+use Phpcq\Repository\RepositoryInterface;
+
+trait InstalledRepositoryLoadingCommandTrait
+{
+    private function getInstalledRepository(string $phpcqPath): RepositoryInterface
+    {
+        if (!is_file($phpcqPath . '/installed.json')) {
+            throw new RuntimeException('Please install the tools first ("phpcq update").');
+        }
+        $loader = new InstalledRepositoryLoader(PlatformInformation::createFromCurrentPlatform());
+
+        return $loader->loadFile($phpcqPath . '/installed.json');
+    }
+}
```

Comment:
I'm thinking about it a bit longer. Should we introduce a central service factory (e.g. [phive has one](https://github.com/phar-io/phive/blob/master/src/Factory.php)) / [self written service container](https://matthiasnoback.nl/2019/03/hand-written-service-containers/) to create instances or is it too over engineered for phpcq?